drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/exporters/cde_export.rb

```diff
@@ -0,0 +1,64 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+module HmisExternalApis::AcHmis::Exporters
```

Comment:
There is a bit of repeated code across these Exporters, for example the `initialize` and `write_row` methods. Would it be good to DRY them up a bit with a parent class or with a Concern?

Reply:
Yes I think a concern would be ideal!